OpenRA.Mods.Common/Activities/Move/Move.cs

```diff
@@ -210,6 +210,13 @@ public override bool Tick(Actor self)
 
 			var nextCell = path[path.Count - 1];
 
+			// Something else might have moved us, so the path is no longer valid.
+			if (!Util.AdjacentCells(self.World, Target.FromCell(self.World, mobile.ToCell)).Contains(nextCell))
```

Comment:
Can we please define
```csharp
public static bool AreNeighbours(CPos a, CPos b)
{
	var offset = b - a;
	return Math.Abs(offset.X) < 2 && Math.Abs(offset.Y) < 2;
}
```

after `Neighbours` in `OpenRA.Mods.Common.Util` and change this line to
```csharp
if (!Util.AreNeighbours(mobile.ToCell, nextCell))
```
?

The current line does a significant amount of work to achieve the same result.

Reply:
Done.